app/assets/stylesheets/active_admin.scss

```diff
@@ -18,6 +18,7 @@ $panelHeaderBck: #002744;
 
 @import "active_skin";
 
+$otherColor: #FFF;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected "#FFF" to be "#fff"         color-hex-case